script-library/docs/docker-in-docker.md

```diff
@@ -28,14 +28,14 @@
 
 See the [`docker-in-docker`](../../containers/docker-in-docker) definition for a complete working example. However, here are the general steps to use the script:
 
-1. Add [`docker-debian.sh`](../docker-debian.sh) to `.devcontainer/library-scripts`
+1. Add [`docker-debian.sh`](../docker-debian.sh) to `.devcontainer/library-scripts`. Make this file executable: `chmod +x .devcontainer/library-scripts/docker-debian.sh`
```

Comment:
Since bash is called directly in the snippet below, this is technically not required.

That said,  this references the wrong script. 😦